src/System.Management.Automation/engine/hostifaces/LocalConnection.cs

```diff
@@ -987,7 +987,7 @@ private void StopOrDisconnectAllJobs()
                 foreach (Job job in this.JobRepository.Jobs)
                 {
                     // Only stop or disconnect PowerShell jobs.
-                    if (job is PSRemotingJob == false)
+                    if ((job is PSRemotingJob) == false)
```

Comment:
I wonder we skipped the compare with false.

Reply:
#13334 will remove the redundant literal